OWNERS_ALIASES

```diff
@@ -22,3 +32,6 @@ aliases:
     - cynepco3hahue
     - fromanirh
     - yuvalk
+    - yrobla           # ZTP focus area
```

Comment:
Prow allows granularity for owners of different parts of a project (See https://kurtmadel.com/posts/native-kubernetes-continuous-delivery/prow/#prow-provides-more-granular-directory-level-permissions-in-github) 

I'd put these in an OWNERS file under `feature-configs/cn-ran-overlays` instead of the root
Same for the reviewers, probably. 
cc @yuvalk @SchSeba

Reply:
+1

Reply:
Thanks for the feedback - I did look at that approach and happy to do the work.  However; I anticipate that the collaboration will be beyond `feature-configs/cn-ran-overlays` I am suggesting a wider collaboration.  I am also working on two PR's to add two more top-level directories.  If we restrict these new contributors to a single directory, we miss the opportunity for wider collaboration.